generate.py

```diff
@@ -39,10 +34,11 @@ async def generate_prompt_response(message, character, context):
     }
     response = requests.post(API_ENDPOINT, headers=headers, json=data)
     print(f"Incoming message from {message.author.display_name}")
+    print(f"Response Status Code: {response.status_code}")  # Print status code

```

Comment:
I'm having second thoughts about this line, but let's leave it for now. Maybe it will be useful.